### PR TITLE
Update DDP docs to include E800 series

### PR DESCRIPTION
**Pull request by martinkennelly**
Update DDP README to add E800 series
Add E800 series examples

### CODE REVIEW

File: docs/ddp/README.md [hunk 1]
```diff
@@ -1,90 +1,76 @@
...
 
+The SR-IOV Network Device plugin can be used to identify currently running DDP *packages*, allowing it to filter Virtual Functions (VFs) by their DDP package name.
 
-# SR-IOV network device plugin with DDP
-Dynamic Device Personalization aka DDP allows dynamic reconfiguration of the packet processing pipeline of Intel Ethernet 700 Series to meet specific use case needs on demand, adding new packet processing pipeline configuration *profiles* to a network adapter at run time, without resetting or rebooting the server.
+In this documentation we will cover the kernel driver use-case only. DPDK configuration of DDP is out of scope.
 
-(ref: [Dynamic Device Personalization for Intel® Ethernet 700 Series](https://software.intel.com/en-us/articles/dynamic-device-personalization-for-intel-ethernet-700-series))
+## Dynamic Device Personalization for Intel® Ethernet Controller E810
+For Intel® Ethernet Controller E810, a DDP package can be loaded into the NIC using the ice kernel driver. Current device DDP state can be determined with DDPTool or devlink.
```
Comment (zshi-redhat) [inline docs/ddp/README.md:9]: Do we want to add devlink support in sriov device plugin?

File: docs/ddp/README.md [hunk 2]
```diff
@@ -1,90 +1,76 @@
...
-$ cp gtp.pkgo /lib/firmware/intel/i40e/ddp/
-```
-
-## 2. Load a DDP profile in to the NIC
-
-Use Linux `ethtool` utility to load a DDP profile into the controller.
-> Note: You can only load DDP profile into a controller using only first Physical Function(PF0).
+#### Intel® Ethernet Controller X710
+Please see the i40e driver readme for full details.
+Use Linux `ethtool` utility to load a DDP package into the controller.
```
[the remark this answers is not in the document]
Comment (martinkennelly) [reply docs/ddp/README.md:50]: @zshi-redhat No reloading of the driver required.

File: docs/ddp/README.md (hunk 1 above)
Comment (martinkennelly) [reply docs/ddp/README.md:9]: @zshi-redhat Yes. It is currently being worked on by Patryk. I discussed it at the last community meeting.

File: docs/ddp/README.md [hunk 3]
```diff
@@ -1,90 +1,76 @@
...
-The latest list of available packages can be found [here]( https://downloadcenter.intel.com/search?keyword=Dynamic+Device+Personalization).
+### Load a DDP package
+#### Intel® Ethernet Controller E810
+##### DDP package applied to a single physical card
+With E810, it is possible to load a different DDP package per physical card. Please see the ice driver readme for full details.
+You must place the DDP package in your NIC firmware folder (typically ```/lib/firmware/updates/intel/ice/ddp/```), append physical card serial number to DDP package name and reload all the physical function drivers on that physical card.
 
-For example, to download and extract the GTP profile:
+##### DDP package applied to all physical cards on a host
+Symbolically link your DDP package to the ```ice.pkg``` package in your NIC firmware folder (typically ```/lib/firmware/updates/intel/ice/ddp```/) and reload all the physical function drivers or simply reload ice driver.
```
[the remark this answers is not in the document]
Comment (martinkennelly) [reply docs/ddp/README.md:46]: @zshi-redhat  Indeed - PF Driver == ice driver. 
There is no difference unless you wanted an E800 card not to be updated for some reason. I will update this.

[the remark this answers is not in the document]
Comment (zshi-redhat) [reply docs/ddp/README.md]: ok, so you actually can unbind the driver from a particular device which doesn't affect other devices using the same driver (guess this is what you mean in the origianl statement?)

[the remark this answers is not in the document]
Comment (martinkennelly) [reply docs/ddp/README.md]: yes exactly.

**Review by zshi-redhat — APPROVED** [review]

**Pull request merged**